web/frontend/src/components/servo.vue

```diff
@@ -23,7 +23,7 @@ const stop = () => {
 const move = (amount: number) => {
   const servo = props.rawStatus;
 
-  const oldAngle = servo.position_deg ?? 0;
+  const oldAngle = servo.positionDeg ?? 0;
```

Comment:
seems like a big change here - was the linter complaining?

Reply:
yes, the linter was complaining - it's possible that the original code is a bug. I will do a smoke test on a servo.

Reply:
@micheal-parks it looks like [the typechecker was wrong for this line before](https://github.com/viamrobotics/rdk/commit/c715fc0afa9f1f2129766e27bd6f74f6dfad05cc#diff-8b55986d4ec45badad0b1a881e045f6d0aba59a0d4f8d0c9e8ebd54741542b3eL26) and we had to override it?

Right now I'm getting an error saying only the camel-cased version of the field exists - is that accurate?

Reply:
@cheukt my smoke test field - we still need to reference `position_deg`. I am adding back [the lint exception that was removed here](https://github.com/viamrobotics/rdk/commit/c715fc0afa9f1f2129766e27bd6f74f6dfad05cc#diff-8b55986d4ec45badad0b1a881e045f6d0aba59a0d4f8d0c9e8ebd54741542b3eL26)